src/NHibernate/Linq/Visitors/TransparentIdentifierRemovingExpressionVisitor.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Remotion.Linq;
+using Remotion.Linq.Clauses.Expressions;
+using Remotion.Linq.Parsing;
+using MemberBinding = Remotion.Linq.Parsing.ExpressionVisitors.MemberBindings.MemberBinding;
+
+namespace NHibernate.Linq.Visitors
+{
+	// Copied from Relinq and added a fallback for comparing two member info by DeclaringType and Name
```

Comment:
Need license and copyright at the top/bottom

Reply:
Added.